### DISCUSSION TITLE
Integration of Prometheus alertmanager and Prometheus with basic auth in domain name

### DISCUSSION THREAD

**Discussion by bhattanil** [prometheus/prometheus#12119] [Q&A]
### What did you do?

I want to integrate alertmanger with Prometheus with domain name and basic auth. How to make this possible. When I put basic auth at Prometheus alert manager it gives error while sending alerts.

### What did you see instead? Under which circumstances?

I see no alerts at alertmanager after putting the auth at alertmanger. What will be the url/ api url to send alerts to alertmanager using basic at alertmanager end?

**Comment by roidelapluie** [comment]
Hi bhattanil, 

Thank you for reaching out. Could you please provide some more details regarding your setup? Such as:

- What version of Prometheus and Alertmanager are you using?
- Could you please share your Prometheus and Alertmanager configuration files?
- What is the error message that you're seeing when trying to send alerts with basic auth?
- What URL are you using to send alerts to Alertmanager?

Once we have this information, we may be able to better assist you with resolving this issue. 

Thank you.

**Reply by bhattanil** [reply]
Hi roidelapluie, 
What version of Prometheus and Alertmanager are you using?
2.40.0-rc.0 / 2022-11-03. Prometheus
Alertmanager version  - 0.24
Could you please share your Prometheus and Alertmanager configuration files?

Default configuration. Sending alerts now to *.*.*.*:9093

What is the error message that you're seeing when trying to send alerts with basic auth?
I am not sure how to use basic auth in Prometheus for alertmanager url so it will use auth details of alertmanager and send alerts to alertmanager 

What URL are you using to send alerts to Alertmanager?
*.*.*.*:9093

Basic requirement is to use auth on both Prometheus and alertmanager. And send alerts to alertmanager. And connect thanos to Prometheus. I am not sure which configuration changes i will do. Or what rest api of prometheus i use in thanossidecar so it will work.

**Reply by bhattanil** [reply]
How to set auth parameters in url.

**Comment by RajeshSamal88** [comment]
Hi Team ,

How to create a ticket with Prometheus team to get a solution in Prometheus integration with Netcool omnibus which we use it as managemnet tool . We are facing issue when we are sending/triggering alert to our management tool but we are getting it to 1 server only not to both .

So we want to check what was the issue .

**Reply by RajeshSamal88** [reply]
alertmanager version - 0.23.0
